README.md

```diff
@@ -4,6 +4,7 @@
 
 ## Important doc links
 
+- [Use CMake Presets](docs/cmake-presets.md)
```

Comment:
I think this looks a little out of place coming before the "quick start" topic.  I think we should move it down one line and call it "Configure a project with CMake Presets".  We can rename the "Configure a project" item to "Configure a project with kits and variants".

Reply:
Fixed.